upstream-sha256: d345391545efc147231b87d54b67971705ff4ab89c099c52242f3db7794bc8fc
actual-sha256: d729b99db28e007cd9a2acc53236fe4ddfc53f08af283b01753f9c5a6d29173b
--- upstream.expected
+++ tolk.actual
@@ -12,7 +12,7 @@
   float val0 = (*(data1_256+(gidx1+alu0)));
   int gidx0 = __ockl_get_group_id(0); /* 16 */
   float val1 = (*(data2_256+(gidx0+alu0)));
-  *(buf2+lidx0) = (val0*val1);
+  *(buf2+lidx0) = (0.0f+(val0*val1));
   __builtin_amdgcn_fence(__ATOMIC_RELEASE, "workgroup");__builtin_amdgcn_s_barrier();__builtin_amdgcn_fence(__ATOMIC_ACQUIRE, "workgroup");
   *(buf1+0) = 0.0f;
   for (int Ridx104 = 0; Ridx104 < 16; Ridx104++) {
